src/transaction-runner.coffee

```diff
@@ -36,16 +36,15 @@ class TransactionRunner
     transactions = if @configuration.options['sorted'] then sortTransactions(transactions) else transactions
 
     logger.verbose('Configuring HTTP transactions')
-    async.mapSeries transactions, @configureTransaction.bind(@), (err, results) =>
-      transactions = results
+    transactions = transactions.map(@configureTransaction.bind(@))
```

Comment:
I made unrelated change - I changed the `configureTransaction()` method to be synchronous (because there was nothing async inside...). It looked like a small change, but then I realised I broke a lot of unit tests 🙈 Anyway, I'm sorry for that. I should have send that separately.